src/pages/home/report/ReportActionItem.tsx

```diff
@@ -274,162 +298,165 @@ function ReportActionItem(props) {
                 CONST.CONTEXT_MENU_TYPES.REPORT_ACTION,
                 event,
                 selection,
-                popoverAnchorRef,
-                props.report.reportID,
-                props.action.reportActionID,
+                popoverAnchorRef.current,
```

Comment:
Why this change? Please test it

Reply:
so ` ReportActionContextMenu.showContextMenu` is typed that we should pass element not a ref object that why it was changed